vignettes/examples/hierarchical_linear_slopes.Rmd

```diff
@@ -0,0 +1,27 @@
+### Random intercept-slope model
+
+A hierarchical, Bayesian linear regression model using the iris data, with random intercepts and slopes for each of the three species.
+The slopes and intercepts for each species are *uncorrelated* in this example.
+
+```{r hierarchical_linear_slopes_greta}
+# linear model parameters
+int <- normal(0, 10)
+coef <- normal(0, 10)
+sd <- cauchy(0, 3, truncation = c(0, Inf))
+
+species_id <- as.numeric(iris$Species)
+
+# random intercepts
+species_random_intercept_sd <- lognormal(0, 1)
```

Comment:
I wonder of these object names could be shortened a bit? I like that they are explicit, but it's a bit unwieldy when they are all combined on a single line.